ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-stable-4.16-upgrade-from-stable-4.12.yaml

```diff
@@ -66,43 +71,47 @@ releases:
     release:
       architecture: amd64
       channel: candidate
-      version: "4.15"
+      version: "4.16"
 resources:
   '*':
     requests:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: azure-ipi-ovn-ipsec-f14
-  cron: 43 3 7,21 * *
+- as: aws-ipi-ovn-ipsec-f14
+  cron: 12 11 7,23 * *
   steps:
-    cluster_profile: azure-qe
+    cluster_profile: aws-qe
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:arm64-target
     env:
-      BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
       OCP_ARCH: arm64
     test:
     - as: set-upgrade-releases
-      commands: echo "${RELEASE_IMAGE_INTERMEDIATE413},${RELEASE_IMAGE_INTERMEDIATE414},${RELEASE_IMAGE_TARGET}"
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE413},${RELEASE_IMAGE_INTERMEDIATE414},${RELEASE_IMAGE_INTERMEDIATE415},${RELEASE_IMAGE_ARM64_TARGET}"
         > ${SHARED_DIR}/upgrade-edge
       dependencies:
-      - env: RELEASE_IMAGE_INTERMEDIATE413
-        name: release:intermediate413
+      - env: RELEASE_IMAGE_INTERMEDIATE415
+        name: release:intermediate415
       - env: RELEASE_IMAGE_INTERMEDIATE414
         name: release:intermediate414
-      - env: RELEASE_IMAGE_TARGET
+      - env: RELEASE_IMAGE_INTERMEDIATE413
+        name: release:intermediate413
+      - env: RELEASE_IMAGE_ARM64_TARGET
         name: release:arm64-target
       from: cli
       resources:
         requests:
           cpu: 100m
           memory: 200Mi
-    - ref: cucushift-chainupgrade-toimage
-    workflow: cucushift-installer-rehearse-azure-ipi-ovn-ipsec
+    - chain: openshift-upgrade-qe-test-chainupgrade
```

Comment:
This is a stable to stable upgrade, so we should use `cucushift-chainupgrade-toimage` here.